appveyor.yml

```diff
@@ -1,19 +1,19 @@
-# Test against the latest version of this Node.js version
-environment:
-  nodejs_version: "8.11.3"
-
-# Install scripts. (runs after repo cloning)
-install:
-  # Get the latest stable version of Node.js or io.js
-  - ps: Install-Product node $env:nodejs_version
-  # install modules
-  - yarn install
-
-# Post-install test scripts.
-test_script:
-  # run tests
-  - yarn lint
-  - yarn test
-
-# Don't actually build.
-build: off
\ No newline at end of file
+# Test against the latest version of this Node.js version
+environment:
+  nodejs_version: "8.11.3"
+
+# Install scripts. (runs after repo cloning)
+install:
+  # Get the latest stable version of Node.js or io.js
+  - ps: Install-Product node $env:nodejs_version
+  # install modules
+  - yarn install
+
+# Post-install test scripts.
+test_script:
+  # run tests
+  - yarn lint
+  - yarn test
+
+# Don't actually build.
+build: false
```

Comment:
How come? It works anyway, and it's using `off` in all our other packages

Reply:
My Webstorm claimed it's not by the schema and I change it automatically. I will revert it